montepy/data_inputs/isotope.py

```diff
@@ -166,8 +166,8 @@ def library(self, library):
             raise TypeError("library must be a string")
         self._library = library
 
-    def __str__(self):
-        return f"{self.element.symbol:>2}-{self.A:<3} ({self._library})"
+    def __repr__(self):
+        return f"{self.__class__.__name__}({repr(self.mcnp_str())})"
```

Comment:
I'm not sure if I like using ZAIDs for `repr`. I would prefer `Isotope('Pu239.80c')`, though this is invalid MCNP.

Reply:
I like that better, too. I'll use the hyph-en-dash variant, because that is valid MCNP 6.3.1

Reply:
[0cdc143](https://github.com/idaholab/MontePy/pull/474/commits/0cdc14374a9df49dcd05307da3fbdef289354154)